flight/Modules/GPS/NMEA.c

```diff
@@ -28,6 +28,8 @@
  * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
  */
 
+#include <math.h>
```

Comment:
Woops.  I wavered with whether I should set it to NAN, so I picked up that line.  Willfix.